package.json

```diff
@@ -16,23 +16,23 @@
     "mini-css-extract-plugin": "2.4.2"
   },
   "dependencies": {
-    "@ckeditor/ckeditor5-clipboard": "34.1.0",
-    "@ckeditor/ckeditor5-core": "34.1.0",
-    "@ckeditor/ckeditor5-engine": "34.1.0",
+    "@ckeditor/ckeditor5-clipboard": "35.2.1",
+    "@ckeditor/ckeditor5-core": "35.2.1",
+    "@ckeditor/ckeditor5-engine": "35.2.1",
     "@ckeditor/ckeditor5-inspector": "3.0.0",
-    "@ckeditor/ckeditor5-ui": "34.1.0",
-    "@ckeditor/ckeditor5-undo": "34.1.0",
-    "@ckeditor/ckeditor5-utils": "34.1.0",
-    "@ckeditor/ckeditor5-widget": "34.1.0"
+    "@ckeditor/ckeditor5-ui": "35.2.1",
+    "@ckeditor/ckeditor5-undo": "35.2.1",
+    "@ckeditor/ckeditor5-utils": "35.2.1",
+    "@ckeditor/ckeditor5-widget": "35.2.1"
   },
   "devDependencies": {
-    "@ckeditor/ckeditor5-basic-styles": "34.1.0",
-    "@ckeditor/ckeditor5-dev-tests": "30.0.0",
-    "@ckeditor/ckeditor5-editor-classic": "34.1.0",
-    "@ckeditor/ckeditor5-editor-inline": "34.1.0",
-    "@ckeditor/ckeditor5-essentials": "34.1.0",
-    "@ckeditor/ckeditor5-paragraph": "34.1.0",
-    "@ckeditor/ckeditor5-theme-lark": "34.1.0",
```

Comment:
I'm wondering why some deps are defined as `devDependencies` and some as regular `dependencies`

Reply:
Ask upstream 🙂 But yeah, I don't understand it either.